pkg/images/utils/utils_test.go

```diff
@@ -283,3 +283,64 @@ func TestNormalizeImageFullName(t *testing.T) {
 		})
 	}
 }
+
+func TestIsRedHatImage(t *testing.T) {
+	tcs := []struct {
+		name      string
+		imagesStr []string
+		want      bool
+	}{
+		{
+			"images in registries assumed to contain all-redhat images are classified as Red Hat images",
```

Comment:
How about using named field? Also how about unflatten the struct

Reply:
Both done in https://github.com/stackrox/stackrox/pull/15761/commits/72dd89d5dbdbdd49342bbf2250e54cac3b9f694d